### PR TITLE
make the image appear in get details

### PR DESCRIPTION
**Pull request by amanshawar**
update eslint 
change in package json 
make the header title change 
put the image from amazon to get details

### CODE REVIEW

**Review by orjwan98 — APPROVED** [review]

**Review by FidaaHersh — APPROVED** [review]

File: client/src/components/Shared/Form/index.js [hunk 1]
```diff
@@ -18,68 +17,77 @@ import {
 }
   from './form.style';
 
-const Form = () => (
-  <StyledForm>
-    <StyledImgCon>
-      <StyledImg src={img} />
-    </StyledImgCon>
+class Form extends Component {
```
Comment (jema28) [inline client/src/components/Shared/Form/index.js:20]: This doesn't need to be a class component as it has no state.

File: client/src/components/GetDetails/index.js [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
-    <React.Fragment>
-      <Title />
-      <Header />
-      <Form />
-      <Button />
-      <GButton />
-      <Footer />
-    </React.Fragment>
-  );
+class GetDetails extends Component {
```
Comment (jema28) [inline client/src/components/GetDetails/index.js:9]: Same as the other file, if you don't have state you can use a functional stateless component.

File: client/src/components/GetDetails/index.js [hunk 3]
```diff
@@ -1,21 +1,29 @@
...
+    super(props);
+  }
+
+  render() {
+    const image = this.props.location.img;
+    return (
+      <React.Fragment>
+        <Title />
+        <Header title="Get your details" />
+        <Form image={image} />
```
Comment (jema28) [inline client/src/components/GetDetails/index.js]: Have a look in your indentation here to make it clear which elements are children.

File: client/src/components/Shared/Form/index.js [hunk 4]
```diff
@@ -18,68 +17,77 @@ import {
...
+            </StyledSelect>
 
-      </StyledItem>
+            <StyledInput type="text" name="price" placeholder="price" />
 
-    </StyledDiv>
+          </StyledItem>
 
-    <StyledTextarea name="extra" placeholder="More  . e.g. What do you love about it?" />
+        </StyledDiv>
```
Comment (jema28) [inline client/src/components/Shared/Form/index.js:84]: I would call this a styled container instead.

**Review by jema28 — CHANGES_REQUESTED** [review]
Working well, nice work! Have a look at my comments and then ready to merge 👍

**Review by Hadeel-Salamin — APPROVED** [review]

File: client/src/components/GetDetails/index.js (hunk 2 above)
Comment (Hadeel-Salamin) [reply client/src/components/GetDetails/index.js:9]: but we will have state later?

File: client/src/components/GetDetails/index.js (hunk 3 above)
Comment (amanshawar) [reply client/src/components/GetDetails/index.js]: it's good with eslint

File: client/src/components/Shared/Form/index.js (hunk 4 above)
Comment (amanshawar) [reply client/src/components/Shared/Form/index.js:84]: it's text area

**Review by jema28 — APPROVED** [review]
Good to merge 👍

**Pull request merged**